docs_user/modules/proc_adopting-the-loadbalancer-service.adoc

```diff
@@ -5,39 +5,252 @@
+commands convert the existing single CA configuration into a
+dual CA configuration.
++
 [source,bash]
 ----
 include::../../tests/roles/dataplane_adoption/tasks/octavia_certs.yaml[lines="7..83",indent=0]
 ----
 
-These commands convert the existing single CA configuration into a dual CA configuration.
-
-// == TODO: Other things
-// TODO
+. Add the octavia interfaces to each NodeNetworkConfigurationPolicy.
++
+The following command adds the network interface that is being
+used as the VLAN base interface for the management network
+(in this example `enp6s0`) for network isolation on the OpenShift nodes.
+Please replace `enp6s0` with the name of the network interface in your
+OpenShift setup.
++
+[source,bash]
+----
+oc get -n openstack --no-headers nncp | cut -f 1 -d ' ' | while read ; do
+    oc patch -n openstack nncp $REPLY --type=merge --patch '
+    spec:
+      desiredState:
+        interfaces:
+        - description: Octavia vlan host interface
+          name: enp6s0.24
+          state: up
+          type: vlan
+          vlan:
+            base-iface: enp6s0
+            id: 24
+        - bridge:
+            options:
+              stp:
+                enabled: false
+            port:
+            - name: enp6s0.24
+          description: Configuring bridge octbr
+          mtu: 1500
+          name: octbr
+          state: up
+          type: linux-bridge
+    '
+done
+----
 
-== Enabling the {loadbalancer_service} in OpenShift
-Run the following command in order to enable the {loadbalancer_service} CR.
+. Configure the network attachment definition
++
+The `octavia` network attachment is needed to connect pods
+that manage amphorae
+and the OpenvSwitch pods (managed by the OVN operator):
++
+[source,bash]
+----
+$ cat >> octavia-nad.yaml << EOF_CAT
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  labels:
+    osp/net: octavia
+  name: octavia
+  namespace: openstack
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "octavia",
+      "type": "bridge",
+      "bridge": "octbr",
+      "ipam": {
+        "type": "whereabouts",
+        "range": "172.23.0.0/24",
+        "range_start": "172.23.0.30",
+        "range_end": "172.23.0.70",
+        "routes": [
+           {
+             "dst": "172.24.0.0/16",
+             "gw" : "172.23.0.150"
+           }
+         ]
+      }
+    }
+EOF_CAT
+$ oc apply -n openstack -f octavia-nad.yaml
+----
 
+. Enable the {loadbalancer_service} service in OpenShift
++
 [source,bash]
 ----
-$ oc patch openstackcontrolplane openstack --type=merge --patch '
+$ oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
 spec:
+  ovn:
+    template:
+      ovnController:
+        nicMappings:
+          octavia: octbr
   octavia:
     enabled: true
-    template: {}
+    template:
+      apacheContainerImage: registry.redhat.io/rhel8/httpd-24:latest
+      amphoraImageContainerImage: quay.io/gthiemonge/octavia-amphora-image
```

Comment:
I guess we don't have an upstream image yet ;)

Reply:
AFAIK we don't, but I guess for this documentation it is acceptable.